app/api/v2/handlers/health_api.py

```diff
@@ -0,0 +1,30 @@
+import operator
+
+from aiohttp import web
+
+import app
+from app.api.v2 import security
+from app.api.v2.handlers.base_api import BaseApi
+from app.api.v2.schemas.caldera_info import CalderaInfoSchema
+
+
+class HealthApi(BaseApi):
+    def __init__(self, services):
+        super().__init__()
+        self._app_svc = services['app_svc']
```

Comment:
`.get('app_svc')` would be consistent with other parts of CALDERA.

Reply:
I made it a bracketed lookup because it's not an optional dependency (`None` is not a valid value here) so I want to fail if it doesn't exist. 

The fact that it's not optional made me consider moving away from the service locator approach entirely for these api classes and move towards a more formal constructor injection approach but am hesitant to as the service locator approach is found across the codebase.

So. I'm of two minds:

1) Force apis to declare all their dependencies in the initializer:

```python
class HealthApi(...):
    def __init__(self, app_svc):
        super().__init__()
        self._app_svc = app_svc
```

This is appealing to be because it doesn't hide dependencies from callers. At the moment, the initializer doesn't make it know what the class actually depends on.


2) This approach (pass in `services` but fail immediately if it's not found)

This is found throughout the codebase so its an established pattern even if it hides true dependencies from the callers. It also reduces constructor explosion and refactoring if we change up services (e.g., split a service into two).

Thoughts? Regardless, I don't think we want to use `.get()` as it just defers the failure until later in a possibly non-intuitive manner.

Reply:
Either way works for me.